jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -497,6 +497,15 @@ def extract(self, return_bg=False, add_data=True, **kwargs):
         pix_scale_factor = self.aperture_area_along_spectral * self.spectral_cube.meta.get('PIXAR_SR', 1.0)  # noqa
         spec.meta['_pixel_scale_factor'] = pix_scale_factor
 
+        # inform the user if scale factor keyword not in metadata
+        if 'PIXAR_SR' not in self.spectral_cube.meta:
+            snackbar_message = SnackbarMessage(
+                ("PIXAR_SR FITS header keyword not found when parsing spectral cube. "
+                 "Flux/Surface Brightness will use default scale factor of 1."),
+                color="error",
+                sender=self)
+            self.hub.broadcast(snackbar_message)
```

Comment:
this message is slightly inconsistent with the default pixel_scale_factor of 1 used in `coords_info.py`.  Perhaps to be consistent, we should just set `pix_scale_factor = 1` here when PIXAR_SR does not exist?

how will this decision and logic be affected by support for per-pixel surface brightnesses?

Reply:
I believe that in the last sprint review we wanted to display some sort of message if the PIXAR_SR keyword was not in the header of the file loaded letting the user know that the translation would use the default value, and wouldn't really be an accurate translation. I went forward with this approach, and I am not opposed to going in any direction we think best.

Reply:
I still think this is inconsistent though, since the lines above are actually falling back on PIXAR_SR of 1 (although this may become a moot point after the bug @rosteen found is fixed which might entirely remove `aperture_area_along_spectral`).